pages.zh/common/2to3.md

```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> 自动将 Python 2 代码转换成 Python 3。
```

Comment:
```suggestion
> 自动将 Python 2 代码转换成 Python 3.
```

I think this is correct, let other maintainers have a look before committing this since I have never reviewed a Chinese page.

Reply:
I think the `。` is correct here. @marchersimon ?

Reply:
Latin characters after Latin text so it should be `.`.